### ISSUE TITLE
`migration squash` eliminates SQL statements

### ISSUE THREAD

**Issue by zeekrey** [supabase/cli#1618]
**Describe the bug**
I encountered a bug while attempting to squash migrations using ﻿`supabase migration squash. It appears that this operation inadvertently removed SQL statements that targeted the storage schema.

**To Reproduce**
1. Create two migrations and include the following example statement:
```
-- Create buckets
INSERT INTO STORAGE.buckets (id, name)
VALUES (
    'documents',
    'documents'
  );
``
3. Run `supabase migration squash`
4. Check if the above statement is present in the resulting migration file.

**Expected behavior**
I expected that executing ﻿supabase migration squash would not alter the outcome of the migration.

**Desktop (please complete the following information):**
 - OS: macOS
 - Version of CLI: 1.100.1
 - Version of Node.js: 18.13.0

**Comment by sweatybridge** [comment]
related to https://github.com/supabase/cli/issues/1618

**Comment by sagacity-felix** [comment]
It also does not include cron jobs. Perhaps it would be nice to document somewhere in the CLI docs that some statements will be missing. It took us quite some time to figure this out.

**Comment by sweatybridge** [comment]
> Perhaps it would be nice to document somewhere in the CLI docs that some statements will be missing.

Good call. I will update the docs accordingly https://github.com/supabase/cli/pull/2412
